src/test/java/com/jcabi/jdbc/JdbcSessionITCase.java

```diff
@@ -40,6 +46,7 @@
  * Integration case for {@link JdbcSession}.
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
+ * @checkstyle StringLiteralsConcatenationCheck (300 lines)
```

Comment:
@amihaiemil have you seen http://www.yegor256.com/2014/06/19/avoid-string-concatenation.html?